src/models/Component.js

```diff
@@ -339,19 +349,6 @@ class Component extends FileInformation {
     return result;
   }
 
-  /**
-   * Get attributes corresponding to the given definition type.
-   * @param {...string} types - Type of attribute to find.
-   * @returns {ComponentAttribute[]} Component attributes array.
-   */
-  getAttributesByDefinitionType(...types) {
```

Comment:
This removal is place in commit `Add externalId in Component` but I think is not the good commit.

Reply:
done